controllers/account.js

```diff
@@ -16,24 +16,28 @@ router.post('/', (req, res) => {
 		password,
 		passwordNew,
 	} = req.body;
-	const discoverable = !!req.body.discoverable;
+	let discoverable = null;
+	if (req.body.discoverable === 'true') {
+		discoverable = true;
+	}
+	if (req.body.discoverable === 'false') {
+		discoverable = false;
+	}
```

Comment:
shouldn't `discoverable` default to `false`? `null´ is handled like `false` in our code so it is unnecessary to have the second if.

Reply:
nope, we have a nullable boolean here so it has three states, not only two.